apps/web/playwright/managed-event-types.e2e.ts

```diff
@@ -69,15 +71,36 @@ test.describe("Managed Event Types tests", () => {
       await page.goto("/event-types");
       await page.getByTestId("event-types").locator('a[title="managed"]').click();
       await page.getByTestId("vertical-tab-assignment").click();
-      await page.locator('[class$="control"]').filter({ hasText: "Select..." }).click();
+      await page.locator('[aria-label="assignment-dropdown"]').click();
```

Comment:
why not add a `data-testid` on the component and use `getByTestId`  instead

Reply:
i tried passing data-testid to ReactSelect but i couldn't find it the dom tree